apps/o2-blocks/index.php

```diff
@@ -37,11 +37,11 @@
  * Load editor assets.
  */
 function a8c_editor_assets() {
-	$assets = require plugin_dir_path( __FILE__ ) . 'dist/editor.asset.php';
+	$assets = require plugin_dir_path( __FILE__ ) . 'dist/editor.min.asset.php';
```

Comment:
The way ETK avoids this is by manually importing the dependency extraction plugin and setting the name of the generated file like this:

https://github.com/Automattic/wp-calypso/blob/f8077a89921a06a75ba2b93d442f1f2db30ea388/apps/editing-toolkit/webpack.config.js#L90

I think it's easiest for us to just import the .min file instead of dealing with creating a custom dependency extraction plugin instance.